internal/campaigns/executor_test.go

```diff
@@ -170,8 +187,13 @@ func TestExecutor_Integration(t *testing.T) {
 
 				diffsByName := map[string]*diff.FileDiff{}
 				for _, fd := range fileDiffs {
-					diffsByName[fd.OrigName] = fd
+					if fd.NewName == "/dev/null" {
+						diffsByName[fd.OrigName] = fd
```

Comment:
I like this affordance. :+1:

Reply:
👍 goes to @eseliger, I simply typed down what he told me :)